core/models/AccessPoint.js

```diff
@@ -31,6 +31,7 @@ module.exports = {
         },
       },
     },
+    required: ["title"],
```

Comment:
Please never repeat this. Making the title property required is not a part of the task. Moreover, it's not a good idea at all – somebody might want to have just ids for access points.